projects/ccd-case-ui-toolkit/src/lib/shared/commons/case-edit-data/case-edit-data.service.ts

```diff
@@ -51,8 +46,8 @@ export class CaseEditDataService {
     this.formValidationErrors$.next([]);
   }
 
-  public clearCaseLinkError(): void {
-    this.linkError$.next(null);
+  public setLinkedCasesJourneyAtFinalStep(isAtFinalStep): void {
```

Comment:
@johnbenjamin-hmcts - consider adding the argument type to `isAtFinalStep`... I'm guessing it's `boolean`?

Reply:
@Daniel-Lam thank you for pointing this out. I have updated the code now. Yes it is boolean :)